source/_integrations/metoffice.markdown

```diff
@@ -17,22 +17,8 @@ The `metoffice` weather platform uses the Met Office's [DataPoint API](https://w
 
-A number of weather entities are created for each entry created in the configuration by location: one weather entity with a summary of the 3-hourly forecasts and twelve sensor entities for individual reporting on each of the individual measurements. The time supplied for each forecast is the start time for the forecast.
+This integration creates a number of weather entities for each entry created in the configuration by location: one weather entity with a summary and a forecast, and twelve sensor entities for individual reporting on each of the individual measurements, for both 3-hourly and daily updates (to a grand total of 26 entities available). Note that only one of the two summary entities and the 3-hourly sensor entities flagged below are enabled by default, so your system isn't overrun on initial configuration. The time supplied for each forecast is the start time for the forecast.
 
-|Entity|Description|Enabled by default|
-|------|-----------|------------------|
-|weather.met_office_**site name**|Weather entity with state of the current weather condition and attributes of temperature, humidity, wind speed and visibility.|Yes.|
-|sensor.**site name**_feels_like_temperature|Sensor entity giving the current forecast 'feels like' temperature.|No.|
-|sensor.**site name**_humidity|Sensor entity giving the current forecast humidity.|No.|
-|sensor.**site name**_probability_of_precipitation|Sensor entity giving the current forecast chance of rain.|Yes.|
-|sensor.**site name**_station_name|Sensor entity giving the current forecast time interval since midnight UTC.|No.|
-|sensor.**site name**_temperature|Sensor entity giving the current forecast temperature.|Yes.|
-|sensor.**site name**_uv_index|Sensor entity giving the current forecast UV index.|No.|
-|sensor.**site name**_visibility|Sensor entity giving the current forecast visibility classification.|No.|
-|sensor.**site name**_visibility_distance|Sensor entity giving the current forecast visibility distance (as a range).|No.|
-|sensor.**site name**_weather|Sensor entity giving the current forecast weather conditions.|Yes.|
-|sensor.**site name**_wind_direction|Sensor entity giving the current forecast wind direction.|No.|
-|sensor.**site name**_wind_gust|Sensor entity giving the current forecast maximum wind gust.|No.|
-|sensor.**site name**_speed|Sensor entity giving the current forecast wind speed.|Yes.|
+The weather entities are named by concatenating `weather.m̀et_office_` with the site name and the frequency. The available sensor entities; "feels like" temperature, humidity, probability of precipitation, station name, temperature, uv index, visibility, visibility distance, weather, wind direction, wind gust and wind speed are named by concatenating `sensor.` with the site name, the entity and the frequency. Only probability of precipitation, temperature, weather and wind speed are enabled by default.
```

Comment:
So the fixed entity ID is still in here? Do the entities of this integration provide a unique ID?

Reply:
Entities do provide unique IDs. How do you think we should change this wording?